Gradle

ruderphilipp opened the pull request:
siehe https://github.com/willuhn/jameica/pull/32

ruderphilipp committed:
build.gradle erstellt

ruderphilipp committed:
.gitignore für Java-IDEs generiert

willuhn commented:
Hier ebenso wie in Hibiscus mit SWT. Kann man das nicht so einbinden, dass einfach auf Jameica referenziert wird?

ruderphilipp committed:
Gradle: explizite Versionsnummern -> zentraler Katalog

Siehe <https://github.com/ruderphilipp/jmc/commit/6d326d2adc6f8f3f1498a719aa688040f37427ce>

ruderphilipp committed:
DRY: redundante Konfig in zentrale Dateien verschoben
